resolver/GlobalPass.cc

```diff
@@ -132,23 +132,6 @@ void resolveTypeMembers(core::GlobalState &gs, core::SymbolRef sym,
             resolveTypeMember(gs, mixin, tp, sym, typeAliases);
         }
     }
-
-    if (sym.data(gs)->isClassClass()) {
-        for (core::SymbolRef tp : sym.data(gs)->typeMembers()) {
-            auto myVariance = tp.data(gs)->variance();
-            if (myVariance != core::Variance::Invariant) {
-                auto loc = tp.data(gs)->loc();
-                if (!loc.file().data(gs).isPayload()) {
-                    if (auto e = gs.beginError(loc, core::errors::Resolver::VariantTypeMemberInClass)) {
-                        e.setHeader("Classes can only have invariant type members");
```

Comment:
I thought this was an explicit feature that we wanted Sorbet's generics to have. Why have we deleted it? (Maybe I'm missing some context)

Reply:
+1, why was this deleted?

Reply:
In a conversation in slack, I got the impression that you didn't think this was necessary anymore. I'll add it back in.